tests/test_cli/test_get_address.py

```diff
@@ -67,3 +68,43 @@ def test_run_positive(self, *mocks):
             standalone_mode=False,
         )
         self.assertEqual(result.exit_code, 0)
+
+
+class TestGetAddressCommand(AEATestCaseEmpty):
+    """Test 'get-address' command."""
+
+    @classmethod
+    def setup_class(cls) -> None:
+        """
+        Override the 'setup_class' method.
+
+        This will prevent setup of tests at class-level.
+        """
+
+    def setup(self):
+        """Set up the test."""
+        super().setup_class()
+
+    def test_get_address(self, password_or_none):
+        """Run the main test."""
+        self.generate_private_key(password=password_or_none)
+        self.add_private_key(password=password_or_none)
+
+        password_option = ["--password", password_or_none] if password_or_none else []
+        result = self.run_cli_command(
+            "get-address", DEFAULT_LEDGER, *password_option, cwd=self._get_cwd()
+        )
+
+        assert result.exit_code == 0
+
+    def teardown(self) -> None:
+        """Tear down the test."""
+        super().teardown_class()
+
+    @classmethod
+    def teardown_class(cls) -> None:
+        """
+        Override the 'teardown_class' method.
+
+        This will prevent teardown of tests at class-level.
+        """
```

Comment:
@DavidMinarsch @solarw this is a nice pattern we could support in the AEA testing framework. `setup` and `teardown` are called before/after each test; maybe a bit inefficient (a temporary AEA project has to be created each time), but useful for e.g. parameterization as in this case.

Reply:
I think its ok to do here, but if we were to do it on the base class we would conflate the meaning of setup and setup_class. I am not sure that this is desirable.